API/src/main/java/net/juligames/core/api/config/BuildInInterpreters.java

```diff
@@ -46,5 +46,8 @@ public interface BuildInInterpreters {
         return s -> DateFormat.getDateInstance().parse(s);
     }
 
+    @Contract(pure = true)
+    static @NotNull SimpleInterpreter<String> interpreter() {return s -> s;}
```

Comment:
Can be used to maintain the String from the Configuration

Reply:
Why? In the case that you are not familiar with the SimpleInterpreter you can't even use a lambda to create an Interpreter